scripts/zip-projects.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# This script automatically zips up our projects for upload to Github on release.
+# Invoke it like: ./zip-projects.sh <release_version> 
+# It will produce 3 zip files in your current directory which you can upload as a release.
+
+set -e -u -x -o pipefail
+
+function cleanUp() {
+    rm -rf "${TEMP_DIR}"
+}
+
+trap cleanUp EXIT
+
+RELEASE_VERSION=$1
+
+TEMP_DIR=$(mktemp -d || mktemp -d -t tmp)
+
+# Step 1. GDK for Unity repo.
+CORE_REPO="${TEMP_DIR}/core/gdk-for-unity"
```

Comment:
git already does that for us, so either
- skip this step
- remove unnecessary files/folders here as well

Reply:
The idea was to have consistency across the repos - but we could just omit the bundle on the core repo given git does have a source zip.

Also the `*` operator actually globs the local directory, which automatically omits any hidden files/directories - so its unnecessary to remove the folders here as well.